app-tracking-protection/vpn-impl/src/test/java/com/duckduckgo/mobile/android/vpn/integration/NgVpnNetworkStackTest.kt

```diff
@@ -51,6 +52,7 @@ class NgVpnNetworkStackTest {
     private val runtime: Runtime = mock()
     private val appTrackerDetector: AppTrackerDetector = mock()
     private val trackingProtectionAppsRepository: TrackingProtectionAppsRepository = mock()
+    private val appTpFeatureConfig: AppTpFeatureConfig = mock()
```

Comment:
Ditto.
You could mock it but also create a fake, see `FakeNetPRemoteFeatureFactory` for an example